spec/services/hyrax/access_control_list_spec.rb

```diff
@@ -164,4 +164,25 @@
       end
     end
   end
+
+  describe "#destroy" do
+    let(:listener) { Hyrax::Specs::SpyListener.new }
+
+    before do
+      acl << permission
+      acl.save
+
+      # Subscribe to events after acl has been persisted
+      Hyrax.publisher.subscribe(listener)
+    end
+
+    after { Hyrax.publisher.unsubscribe(listener) }
+
+    it 'deletes the acl resource' do
+      expect { acl.destroy }
+        .to change { Hyrax::AccessControl.for(resource: resource, query_service: acl.query_service).persisted? }
```

Comment:
I don't see the listener message defined here, so I'm guessing that the check for the message will happen in another PR?

Reply:
The event and the publishing of it was removed from this PR per discussion on https://github.com/samvera/hyrax/pull/5428